src/core/dbus-util.h

```diff
@@ -179,7 +179,9 @@ int bus_property_get_triggered_unit(sd_bus *bus, const char *path, const char *i
                         sd_bus_error *error) {                          \
                                                                         \
                 const char *s;                                          \
-                type v;                                                 \
+                /* Need to initialize this to avoid a false positive    \
+                 * ‘v’ may be used uninitialized [-Werror=maybe-uninitialized] */ \
+                type v = 0;                                             \
```

Comment:
This should not be false positive. But real bug in the parser.

Reply:
So what, just edit the comment?

Reply:
No no, I mean, it is a bug in your new parser.